### PR TITLE
Add UDP library specification and remove auto-generated files/directories by VSCode

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
1. Resolves https://github.com/ballerina-platform/ballerina-standard-library/issues/2197
2. Add auto-generated files/directories by VSCode into `.gitignore`

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,3 +2,8 @@
 This file contains all the notable changes done to the Ballerina UDP package through the releases.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- [[#2197] Add the specification document](https://github.com/ballerina-platform/ballerina-standard-library/issues/2197)
```
Comment (shafreenAnfar) [inline changelog.md:9]: I think we don't need to update the change log for this.

Comment (Maninda) [reply changelog.md]: Reverted back.

**Review by shafreenAnfar — DISMISSED** [review]

File: .gitignore [hunk 2]
```diff
@@ -37,3 +37,25 @@ target
 # Ballerina
 velocity.log*
 *Ballerina.lock
+
+# VSCode
+.project
```
Comment (ThisaruGuruge) [inline .gitignore:42]: Isn't `**/*.project` work here? Seems redundant to add each and every file in the project.

Comment (Maninda) [reply .gitignore]: Changed along with the other directories by generalizing.

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**